src/Dashboard.tsx

```diff
@@ -1,14 +1,10 @@
 import { useState } from "react";
 import Button from "./components/Button/Button";
 import Input from "./components/Input/Input";
+import { Todo } from "./components/TodoItemList/TodoItemList";
+import TodoItems from "./components/TodoItemList/TodoItemList";
 
-interface Todo {
-  id: number;
-  text: string;
-  completed: boolean;
-}
-
-export const TodoList: React.FC = () => {
+export const Dashboard: React.FC = () => {
```

Comment:
Convert this to the standard we've used in the other files: 

`function component_name() {}` with `export default compnent_name` at the bottom of the file.